examples/preprocessing/plot_fnirs_artifact_removal.py

```diff
@@ -17,7 +17,7 @@
 import mne
 
 from mne.preprocessing.nirs import (
-    optical_density, temporal_derivative_distribution_repair)
+    optical_density, tddr)
```

Comment:
Should be:
```python
from mne.preprocessing.nirs import optical_density, tddr
```

Reply:
Thanks, PyCharm didn't catch this automatically!

Reply:
Well it's not a bug, and it probably complies with PEP8, so there's not much to catch :)

Reply:
Automatic refactoring should be able to take care of such cases IMO, but maybe I'm wrong.